crates/turbopack-node/src/pool.rs

```diff
@@ -105,6 +105,7 @@ impl NodeJsPoolProcess {
         let port = listener.local_addr().context("getting port")?.port();
         let mut cmd = Command::new("node");
         cmd.current_dir(cwd);
+        cmd.arg("--async-stack-traces");
```

Comment:
Was this change meant to be in this PR? Can add a comment to explain why we want this?

Reply:
no should be removed